apps/src/templates/projects/personal-projects-table-actions-cell.module.scss

```
@@ -1,3 +1,7 @@
 .xIcon {
       padding-right: 5px;
 }
+
+.buttonMargin {
+      margin-right: 5px;
```

Can you update this to `8px`? It'll fit better with the 16px grid system we've been implementing 😄
```suggestion
      margin-right: 8px;
```

---

There's a 16px grid system in play! 🤩 This is why I love tagging folks from other teams on PRs. Thanks @kelbyhawn